continuum/scenarios/transformation_incremental.py

```diff
@@ -71,19 +73,51 @@ def __getitem__(self, task_index):
         :param task_index: The unique index of a task, between 0 and len(loader) - 1.
         :return: A train PyTorch's Datasets.
         """
+        x, y, _ = self.dataset
+
         if isinstance(task_index, slice):
-            raise ValueError(
-                "Incremental training based on transformations "
-                "does not support slice, please provide only integer."
-            )
-        elif task_index < 0:  # Support for negative index, e.g. -1 == last
-            while task_index < 0:
-                task_index += len(self)
-
-        self.update_task_indexes(task_index)
-        if not self.shared_label_space:
-            self.update_labels(task_index)
-        x, y, t, _ = self._select_data_by_task(task_index)
-        trsf = self.get_task_transformation(task_index)
+            start = task_index.start if task_index.start is not None else 0
```

Comment:
adding some comments here could help to understand what you are trying to do exactly

Reply:
It's quite straightforward, we simply respect python slicing convention (e.g. if no end then end of list, etc.). But i've added a comment.